bigtable/tests/unit/_testing.py

```diff
@@ -42,5 +51,19 @@ def __call__(self, *args, **kwargs):
         """Sync method meant to mock a gRPC stub request."""
         self._stub.method_calls.append((self._name, args, kwargs))
         curr_result, self._stub.results = (self._stub.results[0],
-                                           self._stub.results[1:])
+                                       self._stub.results[1:])
```

Comment:
This should probably keep the same indentation as in the earlier change (i.e., the `self`s should line up).